changelogs/fragments/no_dot_var.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
+  - replace "." in reference field colums to "_"
```

Comment:
please specify the modified file:
`- now - replace "." in reference field columns to "_"`

Reply:
there's a typo also in `columns`